src/v/cluster/controller_log_limiter.h

```diff
@@ -92,11 +92,10 @@ class controller_log_limiter {
           std::is_same_v<Cmd, delete_acls_cmd>) {
             return _acls_and_users_operations_limiter.try_throttle();
         } else if constexpr (
-          std::is_same_v<Cmd, create_data_policy_cmd> ||    //
-          std::is_same_v<Cmd, delete_data_policy_cmd> ||    //
-          std::is_same_v<Cmd, cluster_config_delta_cmd> ||  //
-          std::is_same_v<Cmd, cluster_config_status_cmd> || //
-          std::is_same_v<Cmd, feature_update_cmd> ||        //
+          std::is_same_v<Cmd, create_data_policy_cmd> ||   //
+          std::is_same_v<Cmd, delete_data_policy_cmd> ||   //
+          std::is_same_v<Cmd, cluster_config_delta_cmd> || //
+          std::is_same_v<Cmd, feature_update_cmd> ||       //
```

Comment:
i think we should not throttle the feature_update command as well

Reply:
I agree